spec/features/donuts/01_user_can_view_a_list_of_donuts_spec.rb

```diff
@@ -11,26 +11,37 @@
 
   let!(:donut) { create(:donut) }
 
-  scenario "sees a list of donuts and a link for a new donut" do
-    visit root_path
+  context "as an unauthenticated user" do
+    scenario "I will be directed to the root and see a list of donuts" do
+      visit root_path
 
-    expect(page).to have_content donut.name
-    expect(page).to have_link donut.name
-
-    click_link "Add New Donut"
-
-    expect(page).to have_content "New Donut Form"
+      expect(page).to have_content "Glazed"
+      expect(page).to_not have_link "Add New Donut"
+    end
   end
 
-  scenario "clicks linked donut name and is taken to show page" do
-    visit root_path
-
-    click_link donut.name
-
-    expect(page).to have_content donut.name
-    expect(page).to have_content donut.description
-    expect(page).to have_content donut.vendor_name
-
-    expect(page).to have_css("img#individual-donut")
+  context "as an authenticated user" do
+    let!(:authenticated_user) { create(:user) }
+
+    before(:each) do
+      login_as(authenticated_user)
+      visit root_path
+    end
+    scenario "an authenticated user is directed to root and sees a list of
```

Comment:
You don't need 'an autheticated user..' part b/c it's part of the context now. Also add a line break to separate this spec